content/lessons/material-data-tables-with-firestore/index.md

```diff
@@ -100,7 +100,7 @@ export class AppModule { }
 
 ## Seeding the Database with Faker
 
-[Faker](https://github.com/marak/Faker.js/) is a handle little tool when you need to seed the database with dummy data. 
+[Faker](https://github.com/faker-js/faker) is a handle little tool when you need to seed the database with dummy data.
```

Comment:
I edited the file with cloud VS Code, provided by the Github platform, which automatically improved the Prettier formatting.
This line includes the actual url change!